index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="./style/main.css">
+    
+    <title>Global Drummers Festival</title>
+</head>
+
+<body class="font">
+    <header class="nav-container">
+        <div class="nav-socials-top">
+            <ul class="nav-socials">
+                <li>
+                    <a href="#"><i class="fa-brands fa-facebook-f"></i> </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="fa-brands fa-twitter"></i>
+                    </a>
+                </li>               
+                <li>
+                    <a href="#">English</a>
+                </li>
+                <li>
+                    <a href="#"> My page</a>
+                </li>
+                <li>
+                    <a href="#">Logout</a>
+                </li>
+            </ul>
+        </div>
+        <nav class="navbar">
+                <a class="logo-container" href="index.html">
+                    <img class="logo" src="./icons/drum-festival-logo.jpg" alt="Drum Festival Logo">
+                </a>       
+                <ul class="navlinks-wrapper">
```

Comment:
- [ ]  As Previous suggested you should position your image and text very well so that it looks similar from provided design . 
![image](https://user-images.githubusercontent.com/43877689/175394633-94402beb-8785-43c2-b3ee-6b398065186c.png)